HelpSource/Classes/Server.schelp

```diff
@@ -153,7 +153,7 @@ method:: sendBundle
 send an OSC-bundle to the server.
 discussion::
 Since the network may have irregular performance, time allows for the bundle to be evaluated at a specified point in the future.
-Thus all messages are synchronous relative to each other, but delayed by a constant offset.
+Thus all messages are synchronous relative to each other but delayed by a constant offset.
```

Comment:
This is more a question of style than correctness, but personally I prefer it with the comma